simpeg_drivers/electricals/direct_current/two_dimensions/constants.py

```diff
@@ -134,6 +128,9 @@
     "vertical_padding": 1000.0,
     "expansion_factor": 1.1,
     "starting_model": 1e-3,
+    "topography_object": UUID("{00000000-0000-0000-0000-000000000000}"),
+    "topography": None,
+    "active_model": None,
     "output_tile_files": False,
     "parallelized": True,
     "n_cpu": None,
```

Comment:
We're missing dependencies on the mesh/mesh creation parameters:
![image](https://github.com/user-attachments/assets/d5df392c-5242-4c87-a7cf-c85074e33b1f)

Reply:
mesh creation should have a 'disabled' dependencyType and conductivity should be 'enabled'

Reply:
Yip